librz/analysis/cc.c

```diff
@@ -7,23 +7,30 @@
 #include <rz_analysis.h>
 #define DB analysis->sdb_cc
 
+#define cc_sdb_unsetf(x, ...) \
+	do { \
+		char *key = rz_str_newf(__VA_ARGS__); \
```

Comment:
Unlikely we will have long names for calling convention, maybe rz_strf() could be used there as well